gen/flights/flight.py

```diff
@@ -172,4 +174,6 @@ def points(self) -> List[FlightWaypoint]:
 
     def __repr__(self):
         name = db.unit_type_name(self.unit_type)
+        if self.custom_name:
```

Comment:
I think if you wanted to make this compatible with 2.3 you could add `custom_name: Optional[str] = None` as a class variable.

I'd probably lean toward not cherry-picking to 2.3 anyway though, at least until we've had it in develop long enough to be sure there are no unintended consequences. Anything that affects debrief has the potential to cause more user pain than most other parts of the code base.

Reply:
Will leave it out of 2.3